www/js/overlay.js

```diff
@@ -309,11 +339,12 @@ const themeSwatchFactory = themeOptions => {
 
 themeSelector.appendChild(h('div.theme-selector-header', 'Presets'));
 
+document.body.classList.toggle('overlay-off', !overlayCast);
```

Comment:
the buttons still work when toggle is off

Reply:
Should be fixed now.